show.go

```diff
@@ -35,9 +35,9 @@ type AggTime struct {
 
 // NewAggTime returns a new AggTime created from a Stream.
 func NewAggTime(stream *Stream, labelFunc func(*Window) string) *AggTime {
-	active := NewBarChart("Active", "App", "Samples", "Time (multiplied by number of windows) application was active")
-	visible := NewBarChart("Visible", "App", "Samples", "Time (multiplied by number of windows) application was visible")
-	all := NewBarChart("All", "App", "Samples", "Time (multiplied by number of windows) application was open")
+	active := NewBarChart("Active", "App", "Samples", "Top 10 active applications by time (multiplied by window count)")
```

Comment:
Welp, updated the strings. 

I suck at go too much to do this the right way. :) If you wanna take a crack, I can merge in your commit or whatever